### PR TITLE
Remove me from rabbitmq-eventing maintainers

### PR DESCRIPTION
**Pull request by Zerpet**
# Changes

- 🗑️ Remove me from the project

/kind removal

Unfortunately, I have to withdraw from the project and focus in other project.

### CODE REVIEW

**Comment by dsimansk** [comment]
@Zerpet sad news. 

To drop yourself from approvers, you will need to remote this entry: https://github.com/knative/community/blob/main/peribolos/knative-extensions.yaml#L1041

and then run `./hack/update-codege.sh` to regen owners files.

File: peribolos/knative.yaml [hunk 1]
```diff
@@ -152,7 +152,6 @@ orgs:
     - yuvalif
     - yuzisun
     - zainabhusain227
-    - Zerpet
```
Comment (dsimansk) [inline peribolos/knative.yaml:155]: This line will effectively remove you from `Knative` gh org, if that's also desired. Then, you want to do the same on knative-extensions.yaml. But that's solely on your decision AFAIK.

Comment (Zerpet) [reply peribolos/knative.yaml:155]: Yeah, this is ok. I don't think I'll get time to contribute to this community any time soon 😔

**Pull request merged**